public/index.html

```diff
@@ -236,6 +236,24 @@ <h3>Layout</h3>
   <docoff-placeholder inline bordered></docoff-placeholder>
 </>
   </textarea>
+
+      <h2><code>docoff-function-doc</code></h2>
+
+      <p><code>&ltdocoff-function-doc&gt;</code> displays function documentation extracted from JSDoc comments in JavaScript or TypeScript files.</p>
+
+      <h3>Usage</h3>
+
+      <h4>Pure HTML</h4>
+      <pre><code>
+&lt;docoff-function-doc href="/path/to/your/functions.js"&gt;&lt;/docoff-function-doc&gt;
+      </code></pre>
+
+      <h3>Example</h3>
+
+      <p>Displaying documentation for example functions:</p>
+
+      <docoff-function-doc href="/exampleJS/functions.js"></docoff-function-doc>
```

Comment:
Lets specify the function to document by adding it after the file name in format `<file/to/path.ts:functionName`

Then there would be only one function per each `docoff-function-doc` definition.